checker/app/model/BaseRule.scala

```diff
@@ -69,6 +73,25 @@ case class RegexRule(
       matcherType = RegexMatcher.getType
     )
   }
+
+  /**
+    * If the first character of the suggestion is identical to the first character
+    * of the matched text case, preserve the original casing. Used when our match covers
+    * the start of a sentence to ensure we don't accidentally lowercase sentence starts.
+    */
+  private def maybePreserveMatchCase(isStartOfSentence: Boolean, suggestion: Suggestion, matchedText: String): Suggestion = suggestion match {
```

Comment:
Any reason this isn't a method on the Suggestion class?

Reply:
Yeah, that's much better, ta.